specifications/api/swagger.yaml

```diff
@@ -1006,6 +1022,14 @@ x-transactionOutputs: &transactionOutputs
       amount: *amount
       assets: *walletAssets
 
+x-transactionNoAmountOutputs: &transactionNoAmountOutputs
```

Comment:
I don't this extra declaration should be necessary.
It's just an array of addresses.

Reply:
so it is used here : 
```
2686     ApiPaymentDestination: &ApiPaymentDestination                                                                                                                                                             
2687       oneOf:                                                                                                                                                                                                  
2688         - title: outputs with specified amounts                                                                                                                                                               
2689           <<: *transactionOutputs                                                                                                                                                                             
2690         - title: outputs without amounts                                                                                                                                                                      
2691           <<: *transactionNoAmountOutputs
```
and in contrast to `transactionOutputs` it does not include `amount` and `assets` as we want to use it in migration. So how to capture migration output here?